x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table_state.tsx

```diff
@@ -78,7 +78,7 @@ export type AlertsTableStateProps = {
   pageSize?: number;
   browserFields?: BrowserFields;
   onUpdate?: (args: TableUpdateHandlerArgs) => void;
-  onLoaded?: () => void;
+  onLoaded?: (alerts: Alerts) => void;
```

Comment:
maybe change this to `onLoaded?: (alerts?: Alerts) => void;` . I'm surprised this is passing ci, since this says 1 argument is required, yet in observability solution it's used with 0.

Reply:
Indeed is surprising. I would say better to leave it as you have it because it will always pass an array of alerts (even empty).

Reply:
@PhilippeOberti , Can't we use existing prop of `onUpdate` because it looks to be doing the same function, I guess? What do you think?

Reply:
Additionally, I would suggest you to consider making the changes in below code instead of changing the Alert Table API. 

This is the hook to render the row actions such as `notes` in the Alert Table. We already have access to all the events here. which can be used to fetch the documents.

https://github.com/elastic/kibana/blob/aca2fdfff64b25dba1b7856385695bd524b70cd7/x-pack/plugins/security_solution/public/detections/hooks/trigger_actions_alert_table/use_actions_column.tsx#L45-L63